src/chrome/content/rules/StyleCaster.com.xml

```diff
@@ -0,0 +1,40 @@
+<!--
+	Connection closed:
+		- thetryonstudio.stylecaster.com
+
+	Connection refused:
+		- videos.stylecaster.com
+
+	Invalid certificate:
+		- cb.stylecaster.com
+		- community.stylecaster.com, equivalent to stylecaster.com
+		- click.email.stylecaster.com
+		- pages.email.stylecaster.com
+		- view.email.stylecaster.com
+		- home.stylecaster.com, equivalent to stylecaster.com
+		- liveintent.stylecaster.com, equivalent to p.liadm.com
+		- news-cdn.stylecaster.com
+		- news.stylecaster.com, equivalent to stylecaster.com
+		- sa-cdn.stylecaster.com
+		- search.stylecaster.com
+		- shop.stylecaster.com
+		- stylestudio.stylecaster.com
+		- ua-cdn.stylecaster.com
```

Comment:
Add returned certificates' Common Names.

Reply:
Added, thanks.

Reply:
@YegorIevlev, please stop saying that. This kind of info serves no real purpose and adds a maintenance burden.

You are driving several contributors away from something that was fine before. I prefer the style shown in the [contributing guide](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts).

For securecookie, see [this discussion](https://github.com/EFForg/https-everywhere/issues/13762) that was weighing towards **NOT** automatically adding a rule for them.

Reply:
@Bisaloo We always used to include this information, but I don't know why it is included.

There is another discussion somewhere, leaning towards using `securecookie` unless there is an actual problem preventing its use. I will try to find it. Currently, there is no standard among contributors. Some include it and some do not.

Reply:
> We always used to include this information, but I don't know why it is included.

That was true a long time ago. As far as I can tell, for years now, reviewers have always ignored it (they don't care one way or the other) or asked to remove it.

Reply:
@Bisaloo For an example of merged PR with CNs, see https://github.com/EFForg/https-everywhere/pull/14928/files. No one asked to remove them.

Reply:
It does not disprove what I said:

> As far as I can tell, for years now, reviewers have always ignored it (they don't care one way or the other) or asked to remove it.

No one asked for this to be added. But you can find examples where reviewers asked for it to be removed.